### ISSUE TITLE
PR Merge Order for new apps

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#66616]
### Description
Under [PR Merge Order](https://depo-platform-documentation.scrollhelp.site/developer-docs/creating-a-new-application#Creatinganewapplication-PRmergeorder), I think the docs should specify that the vets-website merge into main needs to happen _as well_ CI on main needs to complete so that the built assets are available for the content-build CI to pull down. if not content-build will throw a not found error. found this out after watch the content build do its thing for about 45 minutes... :-) 

### Relevant URLs
https://depo-platform-documentation.scrollhelp.site/developer-docs/creating-a-new-application

### Which type of team are you on? (Platform team, VFS team, or Leadership)
VFS team

**Comment by vbellissimo** [comment]
Hi @batemapf, thanks for flagging. We will put on our backlog for next steps.

**Comment by laucon** [comment]
@batemapf hi! Can you tell me, what exactly do you think we should write on [this page](https://depo-platform-documentation.scrollhelp.site/developer-docs/creating-a-new-application) to get this point across? And should we write it under the header, "PR merge order?"

**Comment by batemapf** [comment]
TBH, I'm not the expert on website + content build process. My observation was that not only does your change need to be merged into vets-website before merging your change into content-build, but CI on vets-website needs to finish first (resulting in the built application assets landing in S3...I think) before merging your corresponding change in content-build and kicking off CI over there. But all of this should absolutely be reviewed and confirmed by someone who is an expert at the website + content build process. Could certainly have been misunderstanding what was going on.

**Comment by laucon** [comment]
I'm just letting you know that we are making progress on this. It's just taking some time. :)

**Comment by laucon** [comment]
Rebecca Tolmach figured out what to write and suggested we replace the current text under "PR Merge Order" with this:

`vets-website` and then `content-build`. Whether you use the Yeoman generator or create your application manually, the first thing you'll need to do is get your PR to `vets-website` approved, merged, and post-merge CI on `main` needs to complete so that the build assets are available for `content-build` to pull down. Automated tests should then pass for the registry addition in your `content-build` PR.

Alex Taker's frontend-cop channel then reviewed the text and approved it. I will edit this text in confluence and we will have it deployed Wednesday at 3:30 so long as there are no other issues. :)

**Comment by laucon** [comment]
Thank you, Patrick, for pointing this out. I think a lot of people agreed with you and appreciated your suggestion a lot. :)
